### ISSUE TITLE
HMVC 

### ISSUE THREAD

**Issue by wcervini** [avenirer/codeigniter-matches-cli#5]
Create app, controller, views, or models, on CI-HMVC

**Comment by avenirer** [comment]
never thought of that. What would that entail? Is it only the location of the controllers, views, and models?

**Comment by ssavinel** [comment]
Yes, they might be located in another folder for example `application/modules/[module_name]/controllers`.
The best idea IMHO would seem to add a migration folder inside a module, but as a module cannot know if it is the 2nd, 5th or 42nd migration, I've got no idea on when to apply the migration...

**Comment by opqclick** [comment]
i already installed matches cli via command (php bin/install.php matches-cli master) then i checked files location as the give directory patern from matches cli github repo.all files are located perfectly, then i changed my application/config/config.php file 
$config['uri_protocol'] = 'REQUEST_URI';
to->
$config['uri_protocol'] = 'AUTO';
$config['uri_protocol'] = isset($_SERVER['REQUEST_URI']) ? 'PATH_INFO' : 'CLI';

but (php index.php matches) not working showing (Could not open input file: index.php).

m using (https://github.com/kenjis/codeigniter-composer-installer)

**Comment by avenirer** [comment]
when using cli, you must go to where the main index.php is (the one that instantiates CodeIgniter). and from there use matches.

**Comment by opqclick** [comment]
thanks a lot
